mkdocs.yml

```diff
@@ -7,7 +7,10 @@ plugins:
   - git-revision-date-localized:
       fallback_to_build_date: true
   - search
-  - awesome-pages
+  - awesome-pages:
+        order: asc
```

Comment:
think this is whats causing what is data science to be at the bottom of the nav bar and i dont think we want that

Reply:
ah yeah, so I want the case studies bit to be ordered, but I couldn't figure out how to do it.... I'll remove this and we can sort out the ordering of the case studies/projects in another ticket.